### ISSUE TITLE
(Gradle) Project fails to import into Eclipse when home directory contains links to other filesystems

### ISSUE THREAD

**Issue by philpem** [NationalSecurityAgency/ghidra#499]
**Describe the bug**
If WINE is installed and remote NFS drives are mounted, importing the Ghidra project into Eclipse will take an inordinate amount of time (ten hours) and fails.

This is due to a Ghidra Gradle script searching for `src/main/resources/generic.log4jtest.xml` starting from the user's home directory. This should start searching from the top of Ghidra's Git repository.

The offending lines of code are:
```groovy
// ~/git/ghidra/gradleScripts/test.gradle
def getLogFileUrl() {
	String rootDir = System.properties.get("user.dir")
	String foundFile
	new File(rootDir).eachFileRecurse(groovy.io.FileType.FILES) {
		if (it.path.endsWith('src/main/resources/generic.log4jtest.xml')) {
			foundFile = "file:" + it.path.toString()
			return "file:" + it.path.toString()
		}
	}
	
	return foundFile
}
```

One workaround is to comment out the "rootDir" assignment and supply the path to the Ghidra Git clone:

```
	String rootDir = '/home/philpem/git/ghidra/Ghidra/Framework/Generic/src/main/resources'
```

**To Reproduce**
1. Follow the instructions in DevGuide.md to set up Ghidra for Eclipse.
2. Import Ghidra into Eclipse as a Gradle project.
3. Eclipse proceeds to "Configuring project: decompiler" then angs on "configuring project".

**Expected behavior**
Project is imported and builds normally.

**Screenshots**
```
FAILURE: Build failed with an exception.

* Where:
Script '/home/philpem/git/ghidra/gradleScripts/test.gradle' line: 126

* What went wrong:
A problem occurred evaluating script.
> The provided File object is not a directory: /home/philpem/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/cwd/.wine/dosdevices/z:/proc/self/task/28449/root/proc/3559/task/12471/root/mnt/zfs2/scratch/veryold_homedir/philpem/JUNK/RESTORE/www/temp/misc/originals/originals_archive/file

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

CONFIGURE FAILED in 9h 58m 21s
```

**Environment (please complete the following information):**
 - OS: Ubuntu 18.04 "xenial"
 - Java Version: 11.0
 - Ghidra Version: Git master

**Comment by ghidra4** [comment]
Thanks for reporting.

**Comment by ghidra4** [comment]
The fix will be available in 9.1.  :dragon:
